recipes/example/meta.yaml

```diff
@@ -1,34 +1,44 @@
-{% set version = "1.0" %}
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "3.8.2" %}
 
 package:
-  name: example
-  version: {{ version }}
+    name: simplejson
+    version: {{ version }}
 
 source:
-  git_url: https://github.com/conda-forge/staged-recipes.git
-  git_tag: 24507f6c6b2d2e03caee550e13830df15d791751
+    # It is preferable to get the package "closer" to the source. For example use GitHub instead PyPI whenever possible.
```

Comment:
> We decided recently we can use either.

That is why I added the preferable :smile: 

> We just shouldn't be cloning. So, the tarball is fine.

Hopefully that is clear in the example. I want to avoid adding too much text.

> Should we use an example URL for that here?

I think so. Another example with PyPI might be good. In fact I might change this one to PyPI and create another one that uses `bld.bat` and `build.sh` for a non-python lib from GitHub. That way we cover more ground with two recipes.